MHVLib/HVMobile/HealthVaultService.m

```diff
@@ -324,22 +323,11 @@ - (NSString *)getCastCallInfoSection {
 }
 
 - (void)saveCastCallResults: (NSString *)responseXml {
-
-    @autoreleasepool
-    {
-        @try
-        {
```

Comment:
I think we might want to leave the try/catch here, but add a call to HVASSERT_MESSAGE in the catch so we can get a log if the deserialization fails and throws. At some point, we want to clean up all the try/catch code, but for now it seems safer to leave this one.